src/website/views/components/spinner.pug

```diff
@@ -198,16 +198,31 @@ p.-text
   | Use <a href="../../utilities/text">text utilities</a> such as <code>-text--primary</code>
   | and <code>-text--muted</code> to change the color of any spinner.
 .example.-mb--3
-  .-m--3
-    .a-grid.-align-items--center
+  .a-grid.-noGutter
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap
       each type in ['text--primary', 'text--success', 'text--warning', 'text--danger', 'text--muted']
-        div.-p--2
+        div.-p--2.-text--center
           svg(class=`a-spinner -${type} -md -m--3`,viewBox=`0 0 66 66`)
             title Loading
             circle(class=`path`,cx=`33`,cy=`33`,r=`30`,fill=`none`,stroke-width=`6`)
-            p.-text(class=`-${type}`)
+            p.-text(class=`-${type} -px--1`)
               = `-${type}`
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap(style="background:#000;")
```

Comment:
@davidojedacl can you delete (style="background:#000;") and add -bg--black class?

Reply:
done